examples/rasterizer.py

```diff
@@ -0,0 +1,120 @@
+import taichi as ti
+import numpy as np
+import math
+
+try:
+    ti.init(arch=ti.gpu)
+except:
+    ti.init(arch=ti.cpu)
```

Comment:
```suggestion
ti.init(arch=ti.gpu)
```

Taichi falls back to CPU automatically if no GPU is found :-)